telebot/main.py

```diff
@@ -0,0 +1,37 @@
+import constants as keys # can move the api_key to .env file alternatively
+from telegram.ext import * # install python-telegram-bot using pip3 first
```

Comment:
In general, it is not good practice to use `import *`, instead you should specify which names (functions/variables/classes) that you are importing to prevent unforeseen name conflicts which may be hard to debug. However if the Telegram documentation insists on this method of imports, it would be good to add in that note.